topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -270,21 +280,23 @@ We have codified all of the dependencies you will need into a YAML file that `an
 >
 > 2. Create the `hosts` inventory file if you have not done so, include a group for `[galaxyservers]` with the address of the host where you want to install Galaxy. Remember, if you are running ansible on the same machine as Galaxy will be installed to, you should set `ansible_connection=local`.
 >
->    > ### {% icon question %} Question
->    >
->    > How does your hosts file look?
+>    > > ### {% icon code-in %} Input: Bash
+>    > > ```bash
+>    > > cat hosts
+>    > > ```
+>    > {: .code-in}
 >    >
->    > > ### {% icon solution %} Solution
+>    > > ### {% icon code-out %} Output: Bash
 >    > >
->    > > Your hostname is probably different
+>    > > Your hostname is probably different:
 >    > >
 >    > > ```ini
 >    > > [galaxyservers]
->    > > training-0.example.org ansible_connection=local
+>    > > gat-88.training.galaxyproject.eu ansible_connection=local
 >    > > ```
->    > >
->    > {: .solution}
->    {: .question}
+>    > {: .code-out}
+>    {: .code-io}
```

Comment:
```suggestion
>    {: .code-in}
```

I suppose that's a typo? There's few others of `code-io` in the PR.

Reply:
Nah [intentional](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#code-box). The .code-io wrapping a code-in/out, puts them side by side.

Reply:
Ah, sorry, I should have checked the "docs". It is a bit confusing though :)

Reply:
Oh, definitely. Any suggestions for improving that? We can swap out terminology. code-2col or so?

Reply:
Yep, something along those lines!